brage-import/src/main/java/no/sikt/nva/brage/migration/merger/findexistingpublication/TitleAndTypePublicationFinder.java

```diff
@@ -105,13 +107,26 @@ private Optional<String> fetchResponse(URI uri) {
     }
 
     private URI searchByTypeAndTitleUri(Publication publication) {
+        var additionalQueryParam = getAdditionalQueryParam(publication);
+        var searchUri = getStandardSearchUri(publication);
+        return additionalQueryParam.isPresent()
+                   ? searchUri.addQueryParameter(additionalQueryParam.get().name(), additionalQueryParam.get().value()).getUri()
+                   : searchUri.getUri();
+    }
+
+    private UriWrapper getStandardSearchUri(Publication publication) {
         return UriWrapper.fromHost(apiHost)
                    .addChild(SEARCH)
                    .addChild(RESOURCES)
                    .addQueryParameter(TITLE, getMainTitle(publication).get())
                    .addQueryParameter(CONTEXT_TYPE, getInstanceType(publication).get())
-                   .addQueryParameter(AGGREGATION, NONE)
-                   .getUri();
+                   .addQueryParameter(AGGREGATION, NONE);
+    }
+
+    private Optional<QueryParam> getAdditionalQueryParam(Publication publication) {
+        return publication.getEntityDescription().getReference().getPublicationInstance() instanceof ConferenceReport
+            ? Optional.of(new QueryParam(CONTEXT_TYPE, EVENT))
+            : Optional.empty();
```

Comment:
When publication to merge is of type ConferenceReport, we should search for both ConferenceReports and Events, 
search-api supports search for multiple context types like: `..contextType=ConferenceReport&contextType=Event`